crates/router/src/core/payouts.rs

```diff
@@ -897,10 +897,28 @@ pub async fn payouts_filtered_list_core(
     .map(ForeignFrom::foreign_from)
     .collect();
 
+    let total_count = db
+        .get_total_count_of_filtered_payouts(
+            merchant_account.get_id(),
+            filters.connector.clone(),
+            filters.currency.clone(),
+            filters.status.clone(),
+            filters.payout_method.clone(),
```

Comment:
what about other parameters in `filter`. Are not supposed to apply those filters?

Reply:
No, this is only for the payout specific filters

cc: @sagarnaikjuspay can you confirm?